### PR TITLE
Feature/monitoring

### PR DESCRIPTION
**Pull request by jamesli9572**
ticket [link](https://www.notion.so/Monitoring-Agenth-eccc57000f3749ad920fc247b8eb5885)
adding script to gather host info and usage

### CODE REVIEW

File: linux_sql/scripts/host_info.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+total_mem=$(vmstat --unit M | tail -1 | awk '{print $4}')
+timestamp=$(date --rfc-3339=seconds | awk -F+ '{print $1}' | xargs)
+
+#constuct insert statement
+insert_statement="INSERT INTO host_info (hostname, cpu_number, cpu_architecture, cpu_model, cpu_mhz, l2_cache, total_mem, "timestamp") VALUES ('${hostname}', ${cpu_number}, '${cpu_architecture}', '${cpu_model}', ${cpu_mhz}, ${L2_cache}, ${total_mem}, '${timestamp}');"
+echo $insert_statement
+
+#execute insert statement
+#export PGPASSWORD=$psql_password
+#psql -h $psql_host -p $psql_port -U $psql_user -d $db_name -c "$insert_statement"
```
Comment (Jarvis-deprecated-account) [inline linux_sql/scripts/host_info.sh]: why comment out? please fix

**Review by ghost — CHANGES_REQUESTED** [review]

File: linux_sql/scripts/host_info.sh (hunk 1 above)
Comment (jamesli9572) [reply linux_sql/scripts/host_info.sh]: fixed

File: linux_sql/scripts/host_info.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+total_mem=$(cat /proc/meminfo | egrep "^MemTotal:" | awk '{print $2}')
+timestamp=$(date --rfc-3339=seconds | awk -F+ '{print $1}' | xargs)
+
+#constuct insert statement
+insert_statement="INSERT INTO host_info (hostname, cpu_number, cpu_architecture, cpu_model, cpu_mhz, l2_cache, total_mem, "timestamp") VALUES ('${hostname}', ${cpu_number}, '${cpu_architecture}', '${cpu_model}', ${cpu_mhz}, ${L2_cache}, ${total_mem}, '${timestamp}');"
+echo $insert_statement
+
+#execute insert statement
+export PGPASSWORD=$psql_password
+psql -h $psql_host -p $psql_port -U $psql_user -d $db_name -c "$insert_statement"
```
Comment (Jarvis-deprecated-account) [inline linux_sql/scripts/host_info.sh:32]: exit 0

**Review by ghost — CHANGES_REQUESTED** [review]

File: linux_sql/scripts/host_info.sh (hunk 2 above)
Comment (jamesli9572) [reply linux_sql/scripts/host_info.sh:32]: fixed

**Pull request merged**